extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/AbstractJpaOperations.java

```diff
@@ -421,7 +421,8 @@ public boolean exists(Class<?> entityClass, String query, Parameters params) {
     }
 
     public long deleteAll(Class<?> entityClass) {
-        return (long) getEntityManager(entityClass).createQuery("DELETE FROM " + getEntityName(entityClass)).executeUpdate();
+        return (long) getEntityManager().createQuery("DELETE FROM " + getEntityName(entityClass)).executeUpdate();
```

Comment:
This doesn't seem right TBH

Reply:
Weird, I should have done a wrong copy/paste or something. Fixed in the last push